### DISCUSSION TITLE
Does crates.io how to monitor status about crate.io service or doc.rs service?

### DISCUSSION THREAD

**Discussion by baoyachi** [rust-lang/crates.io#6227] [Q&A]
So, where can I find the source code for this website [https://status.crates.io/](https://status.crates.io/) ?

<img width="1187" alt="image" src="https://user-images.githubusercontent.com/10433001/226976625-a84051ed-dd28-43fc-aff6-c3f118452693.png">

**Comment by Turbo87** [comment]
that page is an external service which is not open source afaik
